app/onepassword_events_api/appserver/static/javascript/views/setup_page.js

```diff
@@ -104,6 +107,12 @@ export async function perform(
     splunk_js_sdk_service,
     ITEMUSAAGE_INPUT,
     { disabled: 1 },
+    false
+  );
+  await updateInputs(
+    splunk_js_sdk_service,
+    AUDITEVENTS_INPUT,
+    { disabled: 1 },
     true
```

Comment:
should this be `false`? like the ones above?

Reply:
no, the last one in the group should be `true` after disabling everything to reload everything all at once. The item usage one used to had `true` here, but audit events is now the last one so audit events have `true` here and item usages have `false` there.

Reply:
totally missed that. Got it, thank you!